dataadaptertest/dataadaptertest.go

```diff
@@ -48,7 +48,7 @@ type TestAdapterInfo struct {
 func GetName(adapterInfo *TestAdapterInfo) (err error) {
 	name := adapterInfo.Adapter.GetName()
 	if name != adapterInfo.Name {
-		return fmt.Errorf("wrong adapter %s name: %s", adapterInfo.Name, name)
+		return aoserrors.New(fmt.Sprintf("wrong adapter %s name: %s", adapterInfo.Name, name))
```

Comment:
Here and following use aoserrors.Errorf for formatted error.

Reply:
fixed